frontend/app/scripts/collections/cars.js

```diff
@@ -27,6 +40,16 @@ define([
       return _.each(items, function(item) {
         item.companyLogo = 'img/' + item.app + '.png';
       });
+    },
+
+    // Resets cached flag after a particular
+    // threshold has passed by
+    _resetCached: function() {
```

Comment:
Change name to method to communicate intent. This method is doing more than reseting the ``cached`` flag.